### PR TITLE
Update install section of documentation

### PR DESCRIPTION
**Pull request by terrymyc**
This PR updates the install section of the documentation. The [installation guide on the website](https://asreview.nl/download) should link to the documentation showing Docker and server installation options.

Questions:
- Is the issue with M1-chipped Apple computers still there?
- Linux is supported, but no installation guide for Linux users on the website.

Todo:
- [x] server installation docs
- [x] style troubleshooting section

image

image

image

image

image

### CODE REVIEW

**Comment by J535D165** [comment]
> * Is the issue with M1-chipped Apple computers still there?

Yes. Not 100% sure, but I think there are still some issues. 

> * Linux is supported, but no installation guide for Linux users on the website.

https://asreview.nl/download/ there is!

File: docs/source/install/docker.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<https://hub.docker.com/r/asreview/asreview>`__ Docker Hub repository.
+
+Open `Terminal
+<https://support.apple.com/guide/terminal/open-or-quit-terminal-apd5265185d-f365-44cb-8b09-71a064a42125/mac>`__
+(macOS) or `Command Prompt
+<https://www.lifewire.com/how-to-open-command-prompt-2618089>`__ (Windows) and enter:
+
+.. code-block:: bash
+
+    docker pull asreview/asreview  # latest stable release
```
Comment (J535D165) [inline docs/source/install/docker.rst:25]: No need to pull, isn't it? You can skip this. See https://github.com/asreview/asreview/tree/master/docker#quickstart-using-docker

File: docs/source/install/install.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Install ASReview LAB
+====================
+
+ASReview LAB is supported on Windows and macOS.
```
Comment (J535D165) [inline docs/source/install/install.rst:4]: Linus is also supported. Download stats show quite some users!

`macOS` -> `MacOSX`

Comment (terrymyc) [reply docs/source/install/install.rst:4]: Isn't `Mac OS X` an outdated name? https://en.wikipedia.org/wiki/MacOS

File: docs/source/install/docker.rst (hunk 1 above)
Comment (terrymyc) [reply docs/source/install/docker.rst:25]: Indeed, if the image is not installed, `docker run` will pull anyway. My question is: maybe users should be aware of this download step? The downloaded image is quite a large file.

**Comment by terrymyc** [comment]
@J535D165 The installation guide for Linux is the same as that for macOS on the website, but Miniconda has Linux installers. Should we separate them?

**Comment by J535D165** [comment]
Most Linux distros have Python installed. And the users are probably familiar with installing Python packages. I dont think we need any action for the website atm.

File: docs/source/install/docker.rst (hunk 1 above)
Comment (J535D165) [reply docs/source/install/docker.rst:25]: Docker users will probably know. They can read about Docker when installing Docker or from the various resources online.

**Comment by terrymyc** [comment]
@J535D165 The issue #738 and the workaround are based on installing ASReview LAB in DEVELOPMENT mode. Is it an issue with doing `pip install asreview`?

**Comment by J535D165** [comment]
Oef, yes this is super tricky now development is merged. This requires a hotfix to the docs and website indeed.

**Comment by terrymyc** [comment]
> Oef, yes this is super tricky now development is merged. This requires a hotfix to the docs and website indeed.

I mean the issue states that the installation from the local git repo by `pip install -e .` will encounter the issue. A [recent comment](https://github.com/asreview/asreview/issues/738#issuecomment-1072085716) mentioned `pip install asreview` worked.

**Comment by J535D165** [comment]
> I mean the issue states that the installation from the local git repo by pip install -e . will encounter the issue.

Yes I know. That's not what we want. 

I just tried to reproduce it on my M1, and still having the issue with sklearn (the main problem).

**Comment by terrymyc** [comment]
@J535D165 Another question: we instruct users to check the Python version by `python --version`, but Linux and macOS ship with Python 2 (if I am still right). In this case, `python3 --version` should be used to check the version. Also, `pip3` is used. For example, see [TensorFlow](https://www.tensorflow.org/install/pip#1.-install-the-python-development-environment-on-your-system).

**Comment by terrymyc** [comment]
@J535D165 Is [server installation](https://asreview.readthedocs.io/en/latest/intro/installation.html#server-installation) a part of `Install with pip`?

**Comment by J535D165** [comment]
I dont think so. True, it is usually installed with pip, but you dont want to bother 99% of the users with ports and hosts.

**Comment by terrymyc** [comment]
> I dont think so. True, it is usually installed with pip, but you dont want to bother 99% of the users with ports and hosts.

Makes sense. It can also be installed with Docker on a server?

**Comment by Rensvandeschoot** [comment]
> The [installation guide on the website](https://asreview.nl/download) should link to the documentation showing Docker and server installation options.

Thank you for the reminder! After the release, I will update the website so that the hyperlinks do not get messed up.

**Comment by J535D165** [comment]
Most of your changes are incorporated in #971 now. Thanks for all the valuable suggestions. If there is anything missing, please let me know.

**Pull request closed without merging**